### PR TITLE
Return relative state/storage path

### PR DESCRIPTION
**Pull request by MykolaSolyanko**
Signed-off-by: Mykola Solianko <mykola_solianko@epam.com>

### CODE REVIEW

**Review by Eabramov84 — APPROVED** [review]

File: storagestate/storagestate.go [hunk 1]
```diff
@@ -428,7 +428,7 @@ func (storageState *StorageState) prepareState(
 		return "", aoserrors.Wrap(err)
 	}
 
-	return stateFilePath, nil
+	return path.Base(stateFilePath), nil
```
Comment (al1img) [inline storagestate/storagestate.go:431]: Use https://pkg.go.dev/path/filepath#Rel stateDir

Comment (al1img) [inline storagestate/storagestate.go:431]: Use https://pkg.go.dev/path/filepath#Rel stateDir

File: storagestate/storagestate.go [hunk 2]
```diff
@@ -647,11 +647,11 @@ func (storageState *StorageState) prepareStorage(
 		return "", aoserrors.Wrap(err)
 	}
 
-	if err = os.Chown(storagePath, params.UID, params.GID); err != nil {
+	if err := os.Chown(storagePath, params.UID, params.GID); err != nil {
 		return "", aoserrors.Wrap(err)
 	}
 
-	return storagePath, nil
+	return path.Base(storagePath), nil
```
Comment (al1img) [inline storagestate/storagestate.go:654]: ditto

Comment (MykolaSolyanko) [reply storagestate/storagestate.go]: Done

File: storagestate/storagestate.go (hunk 1 above)
Comment (MykolaSolyanko) [reply storagestate/storagestate.go]: Done

**Review by al1img — APPROVED** [review]

**Pull request merged**